<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcelWgt</class>
 <widget class="QWidget" name="ExcelWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,1,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Excel转换表：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>转换数量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cnt_label">
         <property name="text">
          <string>0/0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,1">
     <item>
      <widget class="QTableWidget" name="table_widget">
       <column>
        <property name="text">
         <string>选择</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>顺序</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>文件</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>表格编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>转换进度</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="select_btn">
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove_btn">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_btn">
           <property name="text">
            <string>重置编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_btn">
           <property name="text">
            <string>保存配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="start_btn">
           <property name="text">
            <string>开始转换</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
